src/Evolution/Systems/ScalarWave/Equations.hpp

```diff
@@ -130,25 +131,41 @@ struct UpwindFlux {
   // clang-tidy: non-const reference
   void pup(PUP::er& /*p*/) noexcept {}  // NOLINT
 
+  // The fields for which the fluxes are computed. There must be a `Variables`
+  // with precisely these fields in the databox.
+  using target_fields = tmpl::list<Pi, Phi<Dim>, Psi>;
+
+  // This is the data needed to compute the numerical flux.
+  // `dg::SendBoundaryFluxes` calls `package_data` to store these tags in a
+  // Variables. Local and remote values of this data are then combined in the
+  // `()` operator.
   using package_tags =
       tmpl::list<Tags::NormalDotFlux<Pi>, Tags::NormalDotFlux<Phi<Dim>>, Pi,
                  NormalTimesFluxPi>;
 
-  using slice_tags = tmpl::list<Pi>;
+  // These tags on the interface of the element are passed to
+  // `package_data` to provide the data needed to compute the numerical fluxes.
+  using argument_tags =
+      tmpl::list<Tags::NormalDotFlux<Pi>, Tags::NormalDotFlux<Phi<Dim>>, Pi>;
```

Comment:
Since we're getting rid of the implicit n.f arguments, let's get rid of the implicit face normal argument as well and put everything into the argument tags.

Reply:
@nilsleiffischer can you make a small PR just adding the comments in this file and the change of `slice_tags`

Reply:
done in #820